core/setup.py

```diff
@@ -71,7 +71,7 @@
         "dbt-extractor~=0.5.0",
         "minimal-snowplow-tracker~=0.0.2",
         "dbt-semantic-interfaces~=0.5.0a2",
-        "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git#egg=dbt",
+        "dbt-common~=0.0.1",
```

Comment:
Updating per semver reocmmendation.
```suggestion
        "dbt-common~=0.1.0",
```